src/main/java/org/jenkinsci/test/acceptance/po/CapybaraPortingLayerImpl.java

```diff
@@ -82,7 +83,23 @@ protected final WebDriver visit(URL url) {
     @Override
     public void clickButton(String text) {
         WebElement e = find(by.button(text));
-        e.click();
+        /*
+         * YUI sticky buttons present some problems when scroll to them, also if you have a sticky button
+         * you should not need to scroll to use them
+         */
+        boolean isStickyButton = false;
+        WebElement stickyContainer = getElement(by.id("bottom-sticker"));
+        if (stickyContainer != null) {
+            JavascriptExecutor je = (JavascriptExecutor)driver;
+            isStickyButton = (boolean)je.executeScript("return arguments[0].contains(arguments[1])", stickyContainer, e);
+        }
+        if (isStickyButton) {
+            Actions builder = new Actions(driver);
+            builder.moveToElement(e).click(e);
```

Comment:
@olivergondza Yes on sticky buttons, on other elements that actually need scrolling I have not tested, but I have created [JENKINS-36551](https://issues.jenkins-ci.org/browse/JENKINS-36551) and I will make a WIP PR to check how does the {{Scroller}} class performs using this method instead of scrollerjs